users/views.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+from rest_framework.authtoken.views import ObtainAuthToken
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from users.models import User, UserRole
+from users.serializers import RegisterSerializer, LoginSerializer
+
+
+class RegisterAPIVIew(APIView):
```

Comment:
`APIVIew` - 'i' z dużej litery. Tak ma być?

Reply:
Nie, nie powinno, to typo. Duży plus za wyłapanie!